vendor/modules.txt

```diff
@@ -113,7 +113,7 @@ github.com/google/uuid
 # github.com/gorilla/mux v1.8.0
 ## explicit; go 1.12
 github.com/gorilla/mux
-# github.com/hpe-storage/common-host-libs v0.0.0-20240717051902-d204e2e2e8fd
+# github.com/hpe-storage/common-host-libs v0.0.0-20240717051902-d204e2e2e8fd => ../common-host-libs
 ## explicit; go 1.19
```

Comment:
The vendor related files should always refer to the github.com path instead of the local path.